openlibrary/templates/type/author/view.html

```diff
@@ -135,8 +135,7 @@ <h2 class="collapse">
                     <input type="hidden" name="sort" value="$query_param('sort')"/>
                   $if (query_param('mode')):
                     <input type="hidden" name="mode" value="$query_param('mode')"/>
-                  <input type="text" placeholder="Search $title Books" name="q" value="$(query_param('q', ''))"/>
-                  <input type="submit"/>
+                  $:render_template("search/searchbox", q=query_param('q'), placeholder='Search {} books'.format(title))
```

Comment:
```suggestion
                  $:render_template("search/searchbox", q=query_param('q'), placeholder=_('Search %(author)s books', author='title'))
```

Reply:
This seems to hardcode the author name as `title` by mistake, will resolve this comment and make a separate commit to fix this.